asyncapi/shopify/service_types.bal

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
```

Comment:
Please add doc comments to explain the usage of this service type. Those doc comments will be displayed in the Choreo UI. Please see the doc comments in this PR - https://github.com/ballerina-platform/asyncapi-triggers/pull/55

Reply:
Resolved in [889f410](https://github.com/ballerina-platform/asyncapi-triggers/pull/70/commits/889f4109ba520dc168db4f9ac9771a534ebec8be). Thank you.